test/problem1_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require_relative '../src/problem1'
+
+describe 'Problem1' do
+
+  array = [2,1,3,3,4]
+  empty_array = []
+  nil_array = nil
+
+  it 'calculate sum' do
```

Comment:
One of the points of `rspec` is to be able to read the specs and make sens of them.

Take a look at this alternative way of writing them, see what happens when a spec fails:

```ruby
describe SumEven do
  subject { SumEven.new.calculate(array) }

  context 'when array is empty' do
    let(:array) { [] }

    # Bonus points if you can explain how this works?
    it { is_expected.to_be zero }
  end

  context 'when array is nil'

  context 'when array is has even numbers' do
    let(:array) { [1, 2, 4, 36, 3 ] }

    it { is_expected.to eq 42 }
  end

  context 'when array has no even numbers'
end
```

_Note: This is an example unrelated to the current spec_